tests/cases/conformance/jsdoc/paramTagOnCallExpression.ts

```diff
@@ -0,0 +1,11 @@
+// @noEmit: true
+// @allowJs: true
+// @checkJs: true
+// @strict: true
```

Comment:
Does this matter?

Reply:
No. It was part of my template when generating a bunch of these similar JSDoc bugs. I removed it.